scripts/bump-version/main.go

```
@@ -0,0 +1,198 @@
+
+	fmt.Fprintf(stdout, "Bumping the version to %s (tag: %s)\n", v, tag)
+	if err := Bump(root, targets, v, stdout); err != nil {
+		return err
+	}
+
+	if err := touch(filepath.Join(root, ".bumptimestamp")); err != nil {
+		return err
+	}
+
+	if !commit && !push {
+		fmt.Fprint(stdout, "\nAll version references were updated and verified. To release, run:\n\n")
```

golangci-lint (2.12.2)

[error] Error return value of `fmt.Fprint` is not checked

(errcheck)